sql1.txt

```diff
@@ -5,25 +5,37 @@ SELECT * FROM Products LIMIT 5;
 
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
-
+SELECT * FROM customers
+ORDER by CustomerName ASC
+LIMIT 10; 
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
-
+SELECT ProductName, Price 
+FROM Products
+WHERE Price < 10;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
-
+SELECT FirstName, LastName
+FROM Employees
+WHERE LastName LIKE "C%";
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
-
+SELECT OrderID, CustomerName 
+FROM Customers 
+JOIN Orders
+ON OrderID = Orders.OrderID
```

Comment:
There's something off with the join columns. The orders table itself only has 196 rows, but your query returned 784, which is too high given that we only want the orders whose customer name starts with `A`.